deploy/olm-catalog/ibm-block-csi-operator-community/1.7.0/ibm-block-csi-operator.v1.7.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,509 @@
+    operatorframework.io/arch.s390x: supported
+    operatorframework.io/arch.ppc64le: supported
+  annotations:
+    capabilities: "Seamless Upgrades"
+    categories: "Storage,Cloud Provider"
+    certified: "true"
+    containerImage: ibmcom/ibm-block-csi-operator:1.7.0
+    createdAt: "2021-06-20T15:00:00Z"
+    description: "Run IBM block storage CSI driver."
+    repository: https://github.com/IBM/ibm-block-csi-operator
+    support: IBM
+    alm-examples: >-
+      [
+        {
+          "apiVersion": "csi.ibm.com/v1",
+          "kind": "IBMBlockCSI",
+          "metadata": {
+            "name": "ibm-block-csi"
+          },
+          "spec": {
+            "controller": {
+              "repository": "ibmcom/ibm-block-csi-driver-controller",
+              "tag": "1.7.0",
+              "imagePullPolicy": "IfNotPresent",
+              "affinity": {
+                "nodeAffinity": {
+                  "requiredDuringSchedulingIgnoredDuringExecution": {
+                    "nodeSelectorTerms": [
+                      {
+                        "matchExpressions": [
+                          {
+                            "key": "kubernetes.io/arch",
+                            "operator": "In",
+                            "values": [
+                              "amd64",
+                              "s390x",
+                              "ppc64le"
+                            ]
+                          }
+                        ]
+                      }
+                    ]
+                  }
+                }
+              }
+            },
+            "node": {
+              "repository": "ibmcom/ibm-block-csi-driver-node",
+              "tag": "1.7.0",
+              "imagePullPolicy": "IfNotPresent",
+              "affinity": {
+                "nodeAffinity": {
+                  "requiredDuringSchedulingIgnoredDuringExecution": {
+                    "nodeSelectorTerms": [
+                      {
+                        "matchExpressions": [
+                          {
+                            "key": "kubernetes.io/arch",
+                            "operator": "In",
+                            "values": [
+                              "amd64",
+                              "s390x",
+                              "ppc64le"
+                            ]
+                          }
+                        ]
+                      }
+                    ]
+                  }
+                }
+              }
+            },
+            "sidecars": [
+              {
+                "name": "csi-node-driver-registrar",
+                "repository": "k8s.gcr.io/sig-storage/csi-node-driver-registrar",
+                "tag": "v2.1.0",
+                "imagePullPolicy": "IfNotPresent"
+              },
+              {
+                "name": "csi-provisioner",
+                "repository": "k8s.gcr.io/sig-storage/csi-provisioner",
+                "tag": "v2.1.0",
```

Comment:
is this supposed to be updated [here](https://github.com/IBM/ibm-block-csi-operator/pull/179#pullrequestreview-685721221)? if so, please change it to draft, until it's ready

Reply:
Done